test/api/v3/integration/groups/POST-groups_groupId_leave.js

```diff
@@ -0,0 +1,86 @@
+import {
+  generateUser,
+} from '../../../../helpers/api-integration.helper';
+
+describe('POST /groups/:groupId/leave', () => {
+  let user;
+  let userToInvite;
+  let group;
```

Comment:
In the case of tests, it only makes sense to extract the values into variables if it's meaningful to do so. These are just dummy values, so it'd be better to remove them.